src/feeder.ts

```diff
@@ -106,38 +100,35 @@ export default class Feeder {
       ext,
     });
 
-    await fs.writeFile(
+    await fs.writeFileSync(
```

Comment:
Same here. Everything that contains Sync in the name is synchronous so no need to use await. If you do wanna use an asynchronous function prefer fs.promises.writeFile